meta/templates/meta/meta.html

```diff
@@ -45,19 +45,6 @@
         {% if meta.twitter_creator %}{% twitter_prop 'creator' meta.twitter_creator %}{% endif %}
         {% if meta.twitter_site %}{% twitter_prop 'site' meta.twitter_site %}{% endif %}
     {% endif %}
-    {% if meta.use_googleplus %}
-        {% if meta.gplus_author %}<link rel="author" href="https://plus.google.com/{{ meta.gplus_author }}"/>{% endif %}
-        {% if meta.gplus_publisher %}<link rel="publisher" href="https://plus.google.com/{{ meta.gplus_publisher }}"/>{% endif %}
-        {% addtoblock 'html_extra' %}{% googleplus_html_scope meta.gplus_type %}{% endaddtoblock %}
```

Comment:
I think we should keep any propery referencing schema.org, while all links to plus.google.com can go away

I think we can keep the `gplus_` / `googleplus_` misnomer prefix to keep compatibility with the past